README.md

```diff
@@ -48,14 +48,22 @@ There are the following tasks:
   and will place that by default into the `build/vaadin-generated` folder. The folder is
   then later picked up by `jar` and `war` tasks which then package the folder contents properly
   onto the classpath. Note that this task is not automatically hooked into `war`/`jar`/`assemble`/`build` and
-  need to be invoked explicitly.
+  need to be invoked explicitly. WARNING: this task will be skipped if `productionMode` is set to false.
```

Comment:
Should this just be `NOTE:` ?

Reply:
Yeah, it may be just NOTE. I'll update this and add the info regarding the `enableDevServer` also.